podcasts/Whats New/WhatsNewView.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+
+struct WhatsNewView: View {
+    @EnvironmentObject var theme: Theme
+
+    let announcement: WhatsNewAnnouncement
+
+    var body: some View {
+        VStack(spacing: 10) {
+            ZStack(alignment: .top) {
+                Rectangle()
+                    .frame(height: 195)
+
+                HStack {
+                    Spacer()
+                    Button {
+                        NavigationManager.sharedManager.dismissPresentedViewController()
+                    } label: {
+                        ZStack {
+                            Image("close")
+                                .foregroundColor(.white)
+                        }
+                        .frame(width: 44, height: 44)
+                    }
+                }
+            }
+            Text(announcement.title)
+                .font(style: .title3, weight: .bold)
+                .padding(.horizontal)
+                .padding(.top)
+                .foregroundColor(theme.primaryText01)
+            Text(announcement.message)
+                .font(style: .subheadline)
+                .foregroundColor(theme.secondaryText02)
+                .padding(.horizontal)
+                .padding(.bottom)
+            Button("Enable it") {}
+                .buttonStyle(RoundedDarkButton(theme: theme))
+                .padding(.horizontal)
+                .padding(.bottom)
```

Comment:
A small refactoring suggestion: Looking at the `.padding`'s that are being used they essentially equate to a container view having `.padding` on all sides, with the exception of the space between the message/button. 

I think we can clean this up a bit if we wrap it in a VStack and apply the .padding to it:

```suggestion
            VStack(spacing: 10) {
                Text(announcement.title)
                    .font(style: .title3, weight: .bold)
                    .foregroundColor(theme.primaryText01)
                Text(announcement.message)
                    .font(style: .subheadline)
                    .foregroundColor(theme.secondaryText02)
                    .padding(.bottom)
                Button("Enable it") { }
                    .buttonStyle(RoundedDarkButton(theme: theme))
            }
            .padding()
```